civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -3,9 +3,76 @@
 
 /* Welcome page */
 .page-welcome-page {
-  .pane-user-login {
-    max-width: 438px;
-    margin: 0 auto;
-    width: 100%;
-  }
-}
\ No newline at end of file
+    .pane-user-login {
+        max-width: 438px;
+        margin: 0 auto;
+        width: 100%;
+    }
+}
+
+.page-reports {
+    .content {
+        padding: 0;
+        background-color: transparent;
+    }
+
+    .chr_panel__footer {
+        margin-bottom: 20px;
+        margin-right: 20px;
+    }
+}
+
+#reportPivotTable {
```

Comment:
I removed the pivot table style, because it's from another ticket. I'll commit it in another branch.